src/main/java/model/User.java

```diff
@@ -13,6 +17,22 @@ public User(String userId, String password, String name, String email) {
         this.email = email;
     }
 
+    public User(Map<String, String> parameters) {
+        this.userId = parameters.get("userId");
+        this.password = parameters.get("password");
+        this.name = parameters.get("name");
+        this.email = parameters.get("email");
+    }
```

Comment:
```suggestion
    public User(Map<String, String> parameters) {
        this(parameters.get("userId"), parameters.get("password"), parameters.get("name"), parameters.get("email"));
    }
```

생성자가 여러개 필요할 경우 단 하나의 primary constructor 와 다수의 secondary constructor 를 가지도록 디자인하는것을 추천해요. 이렇게 하면 중복 코드를 방지하고 설계를 더 간결하게 만들수 있기 때문이에요. (예를들어 생성자의 입력값 `userId` 를 검증해야 하는 경우를 생각해보면 좋겠어요.)